src/App.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import './App.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { AnimatePresence, motion } from 'framer-motion';
+import Navbarlist from './components/navbarlist/Navbarlist';
+import Home from './components/home/Home';
+import Booking from './components/booking/Booking';
+import Reservation from './components/reservation/Reservations';
+import Detailed from './components/home/detailed/Detailed';
+import DeleteHome from './components/delete_home/DeleteHome';
+import AddHome from './components/add_home/AddHome';
+import Login from './components/login/Login';
+import {
+  loginset,
+  loginreset,
+  usernamereset,
+} from './components/login/loginSlicer';
+import Signup from './components/signup/Signup';
+import './components/home/home.css';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:


![image](https://github.com/asharanjith/stay-a-while-front-end/assets/57421823/2aac6cf2-1766-4c4f-bc8f-50330dae2f38)